code/modules/projectiles/ammunition.dm

```diff
@@ -250,7 +250,12 @@
 
 /obj/item/ammo_box/update_desc()
 	. = ..()
-	desc = "[initial(desc)] There are [length(stored_ammo)] shell\s left!"
+
+	// Edgecase catch to prevent a shell count from appearing on the DL-88 Speedcharger
+	if(istype(src, /obj/item/ammo_box/magazine/detective/speedcharger))
+		desc = "[initial(desc)]"
+	else
+		desc = "[initial(desc)] There are [length(stored_ammo)] shell\s left!"
```

Comment:
This feels a bit hacky to me. I would prefer that a new proc for the DL-88 mags (and any future weapons using this system) be attached to the magazine itself as opposed to snowflake checks in commonly used procs.

For example, adding this proc at line 623 of magazines.dm:
```
/obj/item/ammo_box/magazine/detective/speedcharger/update_desc()
	. = ..()
	desc = "[initial(desc)]"

Reply:
Makes sense, I'll see what I can do